appsembler/gestore
# Add a demo app

## Change description

Add a demo app with various models and views to test the manager on.

> This app is sourced from https://github.com/mdn/django-locallibrary-tutorial


## How to verify
- Start with migrating your DB
```shell
python manage.py migrate
```

- Generate models objects and store them in your local DB by using
```shell
python manage.py generate  # This will create a suppreuser for you with these credentials: `username='admin'` and `password='admin'`
```

- Go to http://127.0.0.1:8000/ and see you objects in action.

> **Note**
> This app is created for the sole purpose of **testing** and **simplifying** the review for the actual commands in #2. No other functionality is expected out of this app.


## Type of change

### Development


### Security


### Code review
